src/state/experiments/selectors.js

```diff
@@ -1,2 +1,29 @@
 export const getExperimentsCount = state =>
   state.get("experiments").get("items").size;
+
+export const getFilteredExperimentsByDate = (state) => {
+  let filteredExperiments = [];
```

Comment:
Ah interesting thing about javascript that isn't exactly intuitive.  `let` and `const` don't actually mean 'this value is mutable' and 'this value is immutable' it just means 'you can reassign to this variable' 'you can't reassign to this variable'.  So this is okay:

    const things = [];
    things.push(thing);

This is not:

    const things = [];
    things = things + [thing];

So do you want filteredExperiments to be `let` or `const`?

Reply:
Ahhh interesting! Ok changed to const